content/blog/2016/2016-11-20-what-jvm-versions-are-running-jenkins-the-return.adoc

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "What JVM versions are running Jenkins? 2016 Update!"
+tags:
+- general
```

Comment:
@rtyler OK, I'll remove, I actually took inspiration from the best :-) https://github.com/jenkins-infra/jenkins.io/blame/master/content/blog/2015/2015-11-03-what-jvm-versions-are-running-jenkins.md#L7